src/Adapter/MSTest.TestAdapter/Resources/Resource.resx

```diff
@@ -384,4 +384,7 @@ Error: {1}</value>
     <comment>{0}: Zero based index if an element inside of an array,
 {1}: Test name</comment>
   </data>
+  <data name="AssemblyResolutionIsOnlyWorkingWithNetFramework" xml:space="preserve">
+    <value>Note that AssemblyResolution is only being applied when using .NET Framework target.</value>
```

Comment:
Maybe we should mention that this is a runsettings configuration entry.

Maybe something like `The runsettings 'AssemblyResolution' entries are only supported in .NET Framework. If your tests are working consider removing this entry otherwise you will need to make sure all dependent assemblies are found in the ouput directory of your test`.

@eleshoe Do you think this warning would give you enough information to understand what to do and how to fix your test?